src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    Scanner scanner = new Scanner(System.in);
+    BetDao dao = new BetDaoImpl();
+
+    public void handle() {
+        String command = scanner.nextLine();
+        int value;
+        double risk;
+        String[] betData;
+        Bet bet;
+        while (true) {
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                betData = command.split(" ");
+                value = Integer.parseInt(betData[0]);
+                risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
```

Comment:
```suggestion
        String command = scanner.nextLine();
        while (true) {
            if (command.equalsIgnoreCase("q")) {
                return;
            }
            try {
                String[] betData = command.split(" ");
                int value = Integer.parseInt(betData[0]);
                double risk = Double.parseDouble(betData[1]);
                Bet bet = new Bet(value, risk);
```

Reply:
Can you explain why this is better?

Reply:
You must declare variables before using.
You don't use them outside the try block